tests/test_cmx_3600_adapter.py

```diff
@@ -415,17 +415,153 @@ def test_nucoda_edl_write(self):
             adapter_name='cmx_3600',
             style='nucoda'
         )
+
+        expected = \
+            'TITLE: test_nucoda_timeline\n\n' \
+            '001  AX       V     C        ' \
+            '00:00:00:00 00:00:00:05 00:00:00:00 00:00:00:05\n' \
+            '* FROM CLIP NAME:  test clip1\n' \
+            '* FROM FILE: S:\\var\\tmp\\test.exr\n' \
+            '002  AX       V     C        ' \
+            '00:00:00:00 00:00:00:05 00:00:00:05 00:00:00:10\n' \
+            '* FROM CLIP NAME:  test clip2\n' \
+            '* FROM FILE: S:\\var\\tmp\\test.exr\n'
```

Comment:
Instead of backslashes you can wrap the whole thing in (), or you could use triple quotes.

Reply:
Much easier to read now. 👍